Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserViewUtils.cpp

```diff
@@ -550,20 +550,22 @@ namespace AzToolsFramework
             }
 
             // Check if this entry has a custom previewer, if so use that thumbnail
-            AZ::EBusAggregateResults<const PreviewerFactory*> factories;
-            PreviewerRequestBus::BroadcastResult(factories, &PreviewerRequests::GetPreviewerFactory, entry);
-            for (const auto factory : factories.values)
+            if (!returnIcon)
```

Comment:
Didn't quite follow what 'returnIcon' mean here and why it is default to false. Would be good to add some comment in the interface.